documentation/modules/snip-mtu-value.adoc

```diff
@@ -0,0 +1,6 @@
+:_content-type: SNIPPET
+
+[NOTE]
+====
+If use any value of MTU besides the default value in your migration network, you must also use the same value in the [ocp-short] transfer network that you use. For more information about the [ocp-short] transfer network, see xref:creating-migration-plan-2-6-3_provider[Creating a migration plan starting with a source provider] or xref:creating-migration-plan-2-6-3_vms[Creating a migration plan starting with specific VMs].
```

Comment:
```suggestion
If you input any value of maximum transmission unit (MTU) besides the default value in your migration network, you must also input the same value in the {ocp-short} transfer network that you use. For more information about the {ocp-short} transfer network, see xref:creating-migration-plan-2-6-3_provider[Creating a migration plan starting with a source provider] or xref:creating-migration-plan-2-6-3_vms[Creating a migration plan starting with specific VMs].
```
1. `If use` (incorrect verb usage or possibly pronoun missing) - change to `If you use` or even `If using`
1. swapping `use` to `input` as you use `use... use` too many times.
1. mis-formatted attributes. please check your previews.

Reply:
Fixed.